conda/cli/main.py

```diff
@@ -37,12 +37,29 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
 import sys
 
+PARSER = None
+
 
 def generate_parser():
+    global PARSER
```

Comment:
I know I know. But we're *especially* sensitive to import times costs right here. I'm not completely happy with it, but at least it's the only `global` in the whole project to my knowledge. Definitely the only one I've put in myself. I probably should have left a comment about why though. Because I would judge too if I came across that randomly.